### PR TITLE
run django checks after migration

### PR DESCRIPTION
**Pull request by snopoke**
see https://github.com/dimagi/commcare-hq/pull/19766

### CODE REVIEW

File: fab/fab/operations/db.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def migrate():
     with cd(env.code_root):
         sudo('%(virtualenv_root)s/bin/python manage.py sync_finish_couchdb_hq' % env)
         sudo('%(virtualenv_root)s/bin/python manage.py migrate_multi --noinput' % env)
+        sudo('%(virtualenv_root)s/bin/python manage.py check --deploy' % env)
```
Comment (esoergel) [inline fab/fab/operations/db.py]: I'm not sure `migrate` is the appropriate place for this, as this could end up checking a bunch of stuff.

Comment (snopoke) [reply fab/fab/operations/db.py]: Yea, agree. I'll move it.

Comment (snopoke) [reply fab/fab/operations/db.py]: 021a381

File: fab/fab/operations/db.py [hunk 2]
```diff
@@ -94,7 +94,6 @@ def migrate():
         sudo('%(virtualenv_root)s/bin/python manage.py sync_finish_couchdb_hq' % env)
         sudo('%(virtualenv_root)s/bin/python manage.py migrate_multi --noinput' % env)
 
-
```
Comment (dannyroberts) [inline fab/fab/operations/db.py]: Nit: this seems like an unintentional/irrelevant change.

**Pull request merged**